charts/openshift-metering/templates/presto/_presto_helpers.tpl

```diff
@@ -45,6 +45,37 @@ connector.name=blackhole
 connector.name=memory
 {{ end }}
 
+{{- define "presto-prometheus-catalog-properties" -}}
+{{- if .Values.presto.spec.config.connectors.prometheus.enabled }}
+{{- with .Values.presto.spec.config.connectors.prometheus -}}
+connector.name=prometheus
+{{- if .uri }}
+prometheus-uri={{ .uri }}
+{{- else }}
+prometheus-uri=http://localhost:9090
+{{- end }}
+{{- if .chunkSizeDuration }}
+query-chunk-size-duration={{ .chunkSizeDuration }}
+{{- else }}
+query-chunk-size-duration=1h
+{{- end }}
+{{- if .maxQueryRangeDuration }}
+max-query-range-duration={{ .maxQueryRangeDuration }}
+{{- else }}
+max-query-range-duration=1d
+{{- end }}
+{{- if .cacheDuration }}
+cache-duration={{ .cacheDuration }}
+{{- else }}
+cache-duration=30s
+{{- end }}
+{{- if .bearer.tokenFile }}
+bearer-token-file={{ .bearer.tokenFile }}
+{{- end }} {{- /* end-if */ -}}
```

Comment:
Hmm, we probably want to update this (and the *-values.yaml) to remove this conditional and use an `emptyDir` to mount the applicable service account when `{{- .useServiceAccountToken }}` is set to true.

Reply:
Done in 21d5c091